google/cloud/bigtable/internal/grpc_error_delegate.cc

```diff
@@ -60,9 +60,9 @@ StatusCode MapStatusCode(grpc::StatusCode const& code) {
       return StatusCode::kUnavailable;
     case grpc::StatusCode::DATA_LOSS:
       return StatusCode::kDataLoss;
-    case grpc::StatusCode::DO_NOT_USE:
     default:
-      return StatusCode::kDoNotUse;
+      std::cerr << "Unexpected grpc::StatusCode: " << code << "\n";
```

Comment:
I added the log because that's what's happening in the ThrowRpcError below. But admittedly that log line immediately precedes a call to Terminate.

This default case should really never be executed in practice. If it ever is, a log doesn't seem bad to me. The options I see are for a should-rarely-if-ever-happen-case are:

1. Log, then do something sane (if doing something sane is possible).
2. Log, then Terminate.
3. Silently return something sane.
4. Silently Terminate.
5.  ???

Thoughts?

Reply:
> 4. Silently return something sane.

Why not `return Status::kUnknown;`?  It represents something like "gRPC returned an unknown status code"... which is exactly the case, right?

Reply:
That is what the code is doing here. That's the "sane" thin the code is currently doing (option 1).

I guess the question is, do we want to also log here to indicate that grpc returned us something that we don't specifically know how to map to anything better than "unknown".

Reply:
It sounds like @devbww and @coryan would prefer to not log to cerr here, which is fine w/ me. Removed the cerr. PTAL.